localnet/common.sh

```
@@ -172,6 +173,13 @@ build-localnet() {
   echo "🤑 Starting LP API ..."
   KEYS_DIR=$KEYS_DIR ./$LOCALNET_INIT_DIR/scripts/start-lp-api.sh $BINARY_ROOT_PATH
 
+  echo "🔬 Starting Deposit Monitor ..."
+  LOCALNET_INIT_DIR=$LOCALNET_INIT_DIR \
```

Shellcheck (0.10.0)

[warning] This assignment is only seen by the forked process.

(SC2097)